backend-rust/src/migrations.rs

```diff
@@ -187,14 +188,16 @@ pub enum SchemaVersion {
     FixDanglingDelegators,
     #[display("0006:Fix staked amounts")]
     FixStakedAmounts,
+    #[display("0007:AccountBaseAddress")]
+    AccountBaseAddress,
 }
 impl SchemaVersion {
     /// The minimum supported database schema version for the API.
     /// Fails at startup if any breaking database schema versions have been
     /// introduced since this version.
-    pub const API_SUPPORTED_SCHEMA_VERSION: SchemaVersion = SchemaVersion::PayDayLotteryPowers;
+    pub const API_SUPPORTED_SCHEMA_VERSION: SchemaVersion = SchemaVersion::FixDanglingDelegators;
```

Comment:
Don't think the API depends on anything in `FixDanglingDelegators`, so it shold still be `PayDayLotteryPowers`.
```suggestion
    pub const API_SUPPORTED_SCHEMA_VERSION: SchemaVersion = SchemaVersion::PayDayLotteryPowers;
```